test.js

```diff
@@ -1,6 +1,6 @@
 import path from 'path';
 import fs from 'fs';
-import stream from 'stream';
+import {Readable} from 'stream';
```

Comment:
`Readable` what? Named imports only makes sense when it's clear enough from the name what it is.

```suggestion
import {Readable as ReadableStream} from 'stream';
```